WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -135,8 +135,8 @@ private extension StoreStatsAndTopPerformersViewController {
             self.syncStats(for: siteID,
                            siteTimezone: timezoneForSync,
                            timeRange: vc.timeRange,
-                           latestDateToInclude: latestDateToInclude) { [weak self] error in
-                if let error = error {
+                           latestDateToInclude: latestDateToInclude) { [weak self] result in
+                if case let .failure(error) = result {
```

Comment:
nit: probably it's better to use a switch with `success` and `failure` here since it's the approach that we used elsewhere.